### PR TITLE
CTDMO-Q metadata cal csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
CTDMO-Q cal csv updates as ID'd during metadata review

### CODE REVIEW

File: calibration/CTDMOQ/CGINS-CTDMOQ-11478__20151224.csv [hunk 1]
```diff
@@ -11,15 +11,15 @@ serial,name,value,notes
...
 37-11478,CC_ptca2,0.01234565,
 37-11478,CC_ptcb0,25.49912,
 37-11478,CC_ptcb1,-0.002375,
 37-11478,CC_ptcb2,0.0,
-37-11478,CC_ptempa0,-70.33879,
+37-11478,CC_ptempa0,-7.033880e+01,
 37-11478,CC_ptempa1,0.05072598,
-37-11478,CC_ptempa2,-4.577423e-07,
+37-11478,CC_ptempa2,-4.577424e-07,
 37-11478,CC_wbotc,3.743861e-07,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/CTDMOQ/CGINS-CTDMOQ-11478__20151224.csv]: Q series - according to Asset Tracking spreadsheet.

**Comment by rtravis-whoi** [comment]
Response to Andrew's comment "calibration/CTDMOQ/CGINS-CTDMOQ-12607__20161027.csv → calibration/CTDMOQ/CGINS-CTDMOQ-12607__20160319.csv should be 20160316 for the cal date":

It actually looks like this should be 20160413 according to the QCT cal (and vendor cal).  Pls verify and then I'll update. (QCT 3305-00101-00350-A.txt)

**Comment by reedan88** [comment]
It should be 20160413. This is why I really don't like reading out-loud. I just zone out.

**Comment by rtravis-whoi** [comment]
Filename has been fixed, Andrew!

File: calibration/CTDMOQ/CGINS-CTDMOQ-11478__20151224.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/CTDMOQ/CGINS-CTDMOQ-11478__20151224.csv]: I must have noticed a couple missing files compared to what was in Vault and added them but forgot to note them on metadata review sheet.  At least the s/n exists in Vault and on the Asset Tracking sheet...so I didn't totally make this up!

**Pull request merged**